test_ros2trace/test/test_ros2trace/test_trace.py

```diff
@@ -309,6 +316,43 @@ def test_default_tracing(self) -> None:
 
         shutil.rmtree(tmpdir)
 
+    @unittest.skipIf(
+        (
+            isinstance(lttngpy.get_tracepoints(domain_type=lttngpy.LTTNG_DOMAIN_KERNEL), int) or
+            isinstance(lttngpy.get_syscalls(), int)
+        ),
+        'kernel tracer is required',
```

Comment:
minor comment.

We would want to print the error number here? this only expects `LTTNG_ERR_KERN_NA` but there are many possible cases that `lttng_list_syscalls` fails. that would be more information when something happens to tell us what to do?

Reply:
You mean you'd like to include the error codes in the user-facing string?

Reply:
can we use `ttngpy.lttng_strerror` to print the error strings from lltng? i think you already use this several places?

Reply:
Absolutely! See the new changes.

Use `--pytest-args -rs` with `colcon test` for it to print the reasons for skipped tests.